changelogs/fragments/_health_check_update_docs.yaml

```diff
@@ -1,9 +1,3 @@
 ---
-doc_changes:
-  - Update examples for ospfv2 resource modules using yaml callback plugin.
-  - Update examples for ospfv3 resource modules using yaml callback plugin.
-  - Update examples for ospf_interfaces resource modules using yaml callback plugin.
-  - Update examples for bgp_global resource modules using yaml callback plugin.
-  - Update examples for bgp_address_family resource modules using yaml callback plugin.
-  - Update examples for bgp_templates resource modules using yaml callback plugin.
-  - Update examples for bgp_neighbor_address_family resource modules using yaml callback plugin.
+minor_changes:
```

Comment:
Looks like you edited an existing changelog fragment. You need to add a new one for your changes.